### PR TITLE
Adding details on how pagination must work

### PR DESCRIPTION
**Pull request by bojan89**

### CODE REVIEW

File: Ad Management API 1.0 FINAL.md [hunk 1]
```diff
@@ -227,6 +228,33 @@ AdCOM objects returned in responses may contain a sparse fieldset to save bandwi
 
 All dates/times must be specified in the format of millis since Unix epoch (January 1, 1970 00:00:00 UTC). If an implementer internally uses a more precise timestamp, values should always be rounded **down** to the nearest millisecond to so that when queries are made using a filter, ads are not missed due to rounding.
 
+## API Pagination <a name="pagination"></a>
+
+Pagination can be very tricky to implement correctly and ensure that all the ad changes can be picked up.
+
+These following edge cases need to be are covered:
```
Comment (ix-jrodriguez) [inline Ad Management API 1.0 FINAL.md:235]: I think this has an extra `are`

File: Ad Management API 1.0 FINAL.md [hunk 2]
```diff
@@ -575,6 +609,7 @@ GET `https://api.superads.com/management/v1/bidder/496/ads?auditStart=1528282813
 {
   "count": 100,
   "more": 1,
+  "nextPage": "https://api.superads.com/management/v1/bidder/496/ads?auditStart=1528306991000_557398",
```
Comment (ix-jrodriguez) [inline Ad Management API 1.0 FINAL.md:612]: Just need to update the format on this one to reflect the use of `timestampId`

Comment (bojan89) [reply Ad Management API 1.0 FINAL.md]: fixed

File: Ad Management API 1.0 FINAL.md (hunk 1 above)
Comment (bojan89) [reply Ad Management API 1.0 FINAL.md]: fixed

**Comment by iantri** [comment]
Thanks @bojan89, @ix-jrodriguez I have folded the material content of this into another branch which I will be merging shortly.

**Pull request closed without merging**